grafana/resource_datasource_permission_test.go

```diff
@@ -72,9 +70,6 @@ func testAccDatasourcePermissionCheckDestroy(datasourceID *int64) resource.TestC
 		if err != nil {
 			return fmt.Errorf("error getting datasource permissions %d: %s", *datasourceID, err)
 		}
-		if response.Enabled {
-			return fmt.Errorf("datasource permissions %d still enabled", *datasourceID)
-		}
```

Comment:
Removing this, as data source permissions are always enabled when RBAC is enabled (which is the default state now).